### PR TITLE
Creating, Editing and Listing Skills

### LINKED ISSUE #4: Page for adding a new Skill
Add a page that allows users to add a new Skill which includes the following fields:

- name
- proficiency
- logo

Collaborate with the fellows contributing to the Python project to understand how logos will be managed. For now, assume this is a string (URL to the image).

When this information is submitted, a POST request should be made to /resume/skill on the server in the following format:

```
{
    "name": "JavaScript",
    "proficiency": "2-4 years",
    "logo": "example-logo.png"
}
```

The server will return a JSON with the position in the list in the following format:

```
{
    "id": 1
}
```

### LINKED ISSUE #12: Page for editing existing Skill
https://github.com/MLH-Fellowship/orientation-project-python-23.SUM.B.1/issues/23

### PR DESCRIPTION
**Pull request by demarillacizere**
Closes #4 
Closes #12

- Created a page for adding a skill
- A page for editing an existing skill
- A section for displaying all skills on the resume
Screenshot 2023-06-29 at 22 13 23

### CODE REVIEW

**Comment by demarillacizere** [comment]
@dtemir

**Comment by demarillacizere** [comment]
@dtemir This PR is ready for review

File: src/routes/skills/edit/id.tsx [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+        body: JSON.stringify({
+          ...data
+        }),
+        method: 'PUT',
+        headers: {
+          'Content-Type': 'application/json'
+        }
+      })
+      
+      return fetch(`${config.VITE_BACKEND_URL}/resume/skill/${id as string}`, {
```
Comment (marcusgchan) [inline src/routes/skills/edit/id.tsx]: I think this is a duplicate put request

Comment (demarillacizere) [reply src/routes/skills/edit/id.tsx]: Thanks for catching that. I've deleted it.

**Pull request merged**